python/BioSimSpace/Parameters/Protocol/_openforcefield.py

```diff
@@ -47,19 +49,32 @@
 # Both Sire and RDKit register the same converter.
 with _warnings.catch_warnings():
     _warnings.filterwarnings("ignore")
-    from rdkit import Chem as _Chem
-    from rdkit import RDLogger as _RDLogger
+    _rdkit = _try_import("rdkit")
+
+    if _have_imported(_rdkit):
+        from rdkit import Chem as _Chem
+        from rdkit import RDLogger as _RDLogger
 
-    # Disable RDKit warnings.
-    _RDLogger.DisableLog('rdApp.*')
+        # Disable RDKit warnings.
+        _RDLogger.DisableLog('rdApp.*')
+    else:
+        _Chem = _rdkit
+        _RDLogger = _rdkit
 
 import sys as _sys
 # Temporarily redirect stderr to suppress import warnings.
 _sys.stderr = open(_os.devnull, "w")
-from simtk.openmm.app import PDBFile as _PDBFile
-from openff.toolkit.topology import Molecule as _OpenFFMolecule
-from openff.toolkit.topology import Topology as _OpenFFTopology
-from openff.toolkit.typing.engines.smirnoff import ForceField as _Forcefield
+
+_openmm = _try_import("openmm")
+
+if _have_imported(_openmm):
+    from openmm.app import PDBFile as _PDBFile
+else:
+    _PDBFile = _openmm
+
+#from openff.toolkit.topology import Molecule as _OpenFFMolecule
+#from openff.toolkit.topology import Topology as _OpenFFTopology
+#from openff.toolkit.typing.engines.smirnoff import ForceField as _Forcefield
```

Comment:
Should these all be commented out? With this in place, we can't parameterise with OpenFF on Linux or macOS, e.g:

```
ParameterisationError: Parameterisation failed! Last error: 'Unable to create OpenFF Molecule!: NameError("name '_OpenFFMolecule' is not defined")'
```

Don't you want to handle `openff-toolkit` in the same way as the other potentially missing dependencies?